service/controller/resource/monitoring/prometheus/resource.go

```diff
@@ -270,54 +270,6 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 		},
 	}
 
-	if config.RemoteWriteURL != "" {
```

Comment:
Along with removing this we can also removing the config and command line flag propagating this value.